docs/build-for-developers/cli-tutorial.md

```diff
@@ -275,34 +183,34 @@ openfn getUsers.js -a http -o tmp/output.json
 ```
 
 <details>
-<summary>See expected CLI logs:</summary>
+  <summary>See expected CLI logs:</summary>
 
-```
-[CLI] ✔ Compiled job from hello.js GET request succeeded with 200 ✓
-[R/T] ✔ Operation 1 complete in 581ms
-[JOB] ℹ {
-  id: 1,
-  name: 'Leanne Graham',
-  username: 'Bret',
-  email: 'Sincere@april.biz',
-  address: {
-    street: 'Kulas Light',
-    suite: 'Apt. 556',
-    city: 'Gwenborough',
-    zipcode: '92998-3874',
-    geo: { lat: '-37.3159', lng: '81.1496' }
-  },
-  phone: '1-770-736-8031 x56442',
-  website: 'hildegard.org',
-  company: {
-    name: 'Romaguera-Crona',
-    catchPhrase: 'Multi-layered client-server neural-net',
-    bs: 'harness real-time e-markets'
+  ```
+  [CLI] ✔ Compiled job from hello.js GET request succeeded with 200 ✓
+  [R/T] ✔ Operation 1 complete in 581ms
+  [JOB] ℹ {
+    id: 1,
+    name: 'Leanne Graham',
```

Comment:
This is fake data right? Can we make that point somewhere? Good to encourage data paranoid amongst our users!

Reply:
It's actual data from [jsonplaceholder](https://jsonplaceholder.typicode.com/).